### PR TITLE
adding api version deprecation check for templates

### PR DESCRIPTION
**Pull request by FreakyGranny**
checks api version in templates

- if version is deprecated - show warning
- if version is upsupported - throw exception

### CODE REVIEW

File: k8s/util.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import logging
```
Comment (rvadim) [inline k8s/util.py]: Lets rename this to deprecationchecker.py or something, not util.py

File: Dockerfile [hunk 2]
```diff
@@ -4,11 +4,20 @@ LABEL description="Image with k8s-handle" \
       maintainer="<Infrastructure & Operations> io@2gis.ru" \
       source="https://github.com/rvadim/k8s-handle"
 
+RUN apk add --no-cache \
```
Comment (rvadim) [inline Dockerfile]: What reason for this change? Bug, simplicity, performance?

File: Dockerfile [hunk 3]
```diff
@@ -4,11 +4,20 @@ LABEL description="Image with k8s-handle" \
...
+        bash \
+        git \
+        ca-certificates \
+        openssl \
+        make
+
+COPY ./requirements.txt /tmp/requirements.txt
```
Comment (rvadim) [inline Dockerfile]: requirements.txt copied in line 18 too

File: Dockerfile (hunk 2 above)
Comment (FreakyGranny) [reply Dockerfile]: re-build of image is faster, because requirements installed in separate layer, just improvement

File: k8s/util.py (hunk 1 above)
Comment (FreakyGranny) [reply k8s/util.py]: ok

File: Dockerfile (hunk 3 above)
Comment (FreakyGranny) [reply Dockerfile]: fixed

**Comment by dekhtyarev** [comment]
LGTm

**Pull request merged**